frontend/src/components/Cases/AccordionItem.js

```diff
@@ -0,0 +1,199 @@
+import AccordionSvg from "./AccordionSvg";
+import { schemaDisplay, copyToClipboard } from "../../utilities/utilFunctions";
+import Svg from "../../assets/svg/Svg";
+
+const AccordionItem = ({ eachCase, implementations, caseImplementation }) => {
+  const seq = eachCase.seq;
+  const description = eachCase.case.description;
+  const schema = eachCase.case.schema;
+  const tests = eachCase.case.tests;
+
+  function result(index, implementation) {
+    var testResult;
+    if (implementation.skipped && implementation.skipped == true) {
+      return (testResult = "skipped");
```

Comment:
What is the reason for assigning value to `testResult` on return?

Reply:
Ya, it's unnecessary I guess. Removed!